../../../pyodbc.pyi,sha256=cLuLanSS0Vry9aUUvn-i1lpDxHiDcMQwQFypGWqD__E,13127
pyodbc-4.0.34.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pyodbc-4.0.34.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.34.dist-info/METADATA,sha256=kHQUDtVcmoqqdj8fr9tl6BnTlS4SL8G9G1iOe7ldeRw,2689
pyodbc-4.0.34.dist-info/RECORD,,
pyodbc-4.0.34.dist-info/WHEEL,sha256=TpFVeXF_cAlV118WSIPWtjqW7nPvzoOw-49FmS3fDKQ,103
pyodbc-4.0.34.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc.cpython-38-x86_64-linux-gnu.so,sha256=41dY4NGqRRBrI9HAhpBfufGwu0SeBCSA2oTLONRU798,921728
